### ISSUE TITLE
Enhance the fuzzy finder methods with exactly matching finder methods

### ISSUE THREAD

**Issue by PeterWippermann** [niloc132/gxt-driver#14]
In general model finder classes use "fuzzy search", i.e. `findElement(By.xpath(".//*[contains(text(),"+escaped+")]"))`.

Examples are:
- org.senchalabs.gwt.gwtdriver.gxt.models.Tree.findItemWithText(String)
- org.senchalabs.gwt.gwtdriver.gxt.models.Button.ButtonFinder.withText(String)
- org.senchalabs.gwt.gwtdriver.gxt.models.Field.FieldFinder.withLabel(String)

In addition to this I find it essential to have exact matchers also. Image a scenario where I have two buttons "new release" and "delete new release". So if I'm trying to find with label text "new release" now, it may happen that I only find the delete button. I don't even have the chance to find the other button then because, you know, I can only search for one item at a time.

I propose to add further methods to the finders, e.g. 
- org.senchalabs.gwt.gwtdriver.gxt.models.Tree.findItemWithExactText(String)
- org.senchalabs.gwt.gwtdriver.gxt.models.Button.ButtonFinder.withExactText(String)
- org.senchalabs.gwt.gwtdriver.gxt.models.Field.FieldFinder.withExactLabel(String)

The implementation of these should use  `findElement(By.xpath(".//*[text()="+escaped+"]"))`

**Comment by niloc132** [comment]
The trick here becomes whitespace, and any content (glyph icons? `::before`/`::after` content?) added by the appearance - I believe the text() will hit those as well, and surprise your test cases. It has been a few years, but I _think_ that is why I implemented all of these with contains() in the first place.

Use of `normalize-space(text())` will at least solve leading/trailing space, but given that newlines/tabs/multiple-spaces in html are rendered as a single space, the tester may be surprised to find that `" "` is the same as `"   "`. 

Worth a shot for sure, with lots of warnings in Javadoc and the like that it must be an _EXACT_ match. (and as with your other issue, this means "patches welcome", feel free to submit something doc'd and tested).

> So if I'm trying to find with label text "new release" now, it may happen that I only find the delete button. I don't even have the chance to find the other button then because, you know, I can only search for one item at a time.

Inside the finder that is true, but you always have the ability to perform a search outside a finder, and then create the widget directly based on the element you found.

**Comment by PeterWippermann** [comment]
Good point @niloc132 -  I sent you an email about contributing some days ago. Would be great if you could briefly answer it, because I'm afraid I'd need to clarify some more details with you.
